packages/ui/src/common/components/statistics/DurationStatistics.tsx

```diff
@@ -29,7 +29,7 @@ export const DurationStatistics = (props: DurationStatisticsProps) => {
   const duration = Date.parse(props.value) - (props.from ? Date.parse(props.from) : Date.now())
   return (
     <StatisticItem {...props}>
-      <DurationValue value={formatDurationDate(duration)} />
+      {duration > 0 ? <DurationValue value={formatDurationDate(duration)} /> : <></>}
```

Comment:
Please resubmit as new PR with only this change:
```suggestion
      {duration && <DurationValue value={formatDurationDate(duration)} />}
```